shell/platform/embedder/embedder.h

```diff
@@ -266,6 +266,15 @@ typedef enum {
 
 typedef struct _FlutterEngine* FLUTTER_API_SYMBOL(FlutterEngine);
 
+/// Identifier for views.
+///
+/// How view IDs are generated is decided by the embedding. The engine does not
+/// assume any specifics about the generation algorithm, including whether it
+/// utilizes the entire range of int64_t or permits the use of zero or negative
+/// IDs. The only requirement is that at any given moment, no two views share
+/// the same ID.
```

Comment:
What do you think of something like this?

```suggestion
/// Unique identifier for views.
///
///
/// View IDs are generated by the embedder and are
/// opaque to the engine. The engine does not interpret view IDs in any way.
```